procrastinate/aiopg_connector.py

```diff
@@ -163,6 +163,8 @@ def set_pool(self, pool: aiopg.Pool) -> None:
         self._pool = pool
 
     async def _get_pool(self) -> aiopg.Pool:
+        if not self._lock:
```

Comment:
Can we put those lines below the other test ? If the pool is manually set, no need to even instantiate the lock.

Reply:
:+1: